sdk/src/manifest_store.rs

```diff
@@ -589,7 +589,13 @@ impl std::fmt::Display for ManifestStore {
         }
 
         json = b64_tag(json, "hash");
+
+        // list of tags to omit (padding tags)
+        // Reason of padding, see note at:
+        // https://c2pa.org/specifications/specifications/2.1/specs/C2PA_Specification.html#_going_back_and_filling_in
         json = omit_tag(json, "pad");
+        json = omit_tag(json, "pad1");
```

Comment:
I really want to rewrite this, each call to omit_tag reparses the entire json string output. so this will double the time it takes.

Reply:
Honestly, with the display cleanup I'm doing, I can remove these calls. It means though that the RAW fallback display will have them too.

Reply:
Those are gone now.